Scripts/script-IsIPInRanges.yml

```diff
@@ -60,3 +60,5 @@ outputs:
 - contextPath: IP.InRange
   description: Is the IP is in the input ranges? (could be 'yes' or 'no)
 scripttarget: 0
+runonce: false
+releaseNotes: "Better handling of IP ranges array"
```

Comment:
I think that this release note is incorrect
At lease this is not what your fix is doing